flake8_quotes/__init__.py

```diff
@@ -92,6 +100,10 @@ def get_quotes_errors(self, file_contents):
                 # ignore strings that do not start with our quotes (both single and multiline)
                 continue
 
+            if self.multiline_ignored and any([token.string.startswith(value) for value in self.multiline_quotes.values()]):
```

Comment:
Doing that causes any tokens with "bad" multiline quotes (even ignored ones) to fail.